lib/eventFactory.js

```diff
@@ -421,7 +421,7 @@ class EventFactory extends BaseFactory {
             // Sync pipeline to make sure workflowGraph is generated
             .then((p) => {
                 // Sync pipeline with the parentEvent sha and create jobs based on that sha
-                if (parentEventId) {
+                if (parentEventId && !prRef) {
```

Comment:
It's ok to check the event is PR or not. In my opinion, the intention looks unclear.

Reply:
I am not clean on PR title vs code change, shouldn't this change be on line 428 below ?

Reply:
What I wanted to say is pipeline sync method with the parent event sha should not be called when PR event.
I will change the title.